src/modules/hit-api.js

```diff
@@ -0,0 +1,15 @@
+export default async function apiData(url = '', data = {}) {
+  const response = await fetch(url, {
+    method: 'POST',
+    mode: 'cors',
+    cache: 'no-cache',
```

Comment:
- [ ] You seem to have addressed the issues from the previous reviewer although you have replaced your arrow functions with regular `function` syntax please make all your functions use the newer ES6 arrow function syntax.
- [ ] Make sure that the project remains working after you make the change